playbooks/remove_node.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: Validate nodes for removal
   hosts: localhost
```

Comment:
Node quite sure why we have localhost here, `run_once: true` would have been a better idea... :thinking: